app/controllers/api/items_controller.rb

```diff
@@ -1,20 +1,4 @@
 class Api::ItemsController < Api::ApiController
-  def sync_manager
-    unless @sync_manager
-      version = params[:api]
-      @sync_manager =
-        if !version
-          SyncEngine::V20161215::SyncManager.new(current_user)
-        elsif version == '20190520'
-          SyncEngine::V20190520::SyncManager.new(current_user)
-        else
-          # Default to latest
-          SyncEngine::V20190520::SyncManager.new(current_user)
-        end
-    end
-    @sync_manager
-  end
```

Comment:
I refactored this method, you can see it below.

From the original method, if no `api` parameter is specified, `SyncEngine::V20161215` is used. Should we change this to a newer version?

Reply:
Yes, should be newet.